api/src/index.js

```diff
@@ -23,9 +25,68 @@ apiRouter.get("/", async (req, res) => {
 
 // This nested router example can also be replaced with your own sub-router
 apiRouter.use("/nested", nestedRouter);
+app.use('/api/meals', mealRouter);
+app.use('/api/reservation', reservationRouter);
+
 
 app.use("/api", apiRouter);
 
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+//Future-meal
+app.get("/future-meals", async (req, res) => {
+  try {
+    const meals = await knex("meal").where("when", ">", knex.fn.now());
+    res.json(meals);
+  } catch (error) {
+    res.status(500).json({ error: error.message });
+  }
+});
+
+//past-meal
+app.get("/past-meals", async (req, res) => {
+  try {
+    const now = new Date();
```

Comment:
here you use:
`const now  = new Date()`

in line 40 you use:
`knex.fn.now()`

Consistency shows that you are a great developer! :trophy: